src/index.ts

```diff
@@ -183,6 +183,29 @@ export default function PluginVue(userOptions: Partial<Options> = {}): Plugin {
         } else if (query.type === 'style') {
           debug(`transform(${id})`)
           const block = descriptor.styles[query.index]!
+
+          let preprocessOptions = options.preprocessOptions || {}
+          const preprocessLang = (options.preprocessStyles
+            ? block.lang
+            : undefined) as SFCAsyncStyleCompileOptions['preprocessLang']
+
+          if (preprocessLang) {
+            preprocessOptions =
+              preprocessOptions[preprocessLang] || preprocessOptions
+
+            if (
+              ['scss', 'sass'].includes(preprocessLang) &&
+              !preprocessOptions.includePaths
+            ) {
+              preprocessOptions = {
+                includePaths: ['node_modules'],
+                ...preprocessOptions,
+              }
```

Comment:
If `node_modules` is included by default for `scss` and `sass`, I think we should add such counterparts  for other `preprocessLang`. AFAIK, Less and Stylus both have a `paths` option.